cloudify_cli/config/parser_config.py

```diff
@@ -400,24 +401,24 @@ def parser_config():
                         'handler': cfy.plugins.download
                     },
                     'list': {
-                        'help': 'List all the plugins on the Manager',
+                        'help': 'List all plugins currently on the Manager',
                         'handler': cfy.plugins.ls
                     },
                     'delete': {
                         'arguments': {
                             '-p,--plugin-id': plugin_id_argument(
-                                hlp='The plugin id'),
+                                hlp="A plugin's ID"),
```

Comment:
Why 'A' and not 'The'?
Notice that in 'cfy snapshots delete', "--snapshot-id"'s help text is:
'The ID of the snapshot to delete'
I suggest that you decide on a format right now, and then go ahead and apply it all over this module =)